src/courseware/course/sidebar/sidebars/notifications/NotificationTray.test.jsx

```diff
@@ -4,8 +4,8 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { breakpoints } from '@openedx/paragon';
 
 import MockAdapter from 'axios-mock-adapter';
-import React from 'react';
 import { Factory } from 'rosie';
+import messages from '../../../messages';
```

Comment:
[question]: Does it make sense to use [aliases](https://github.com/openedx/frontend-app-learning/blob/master/webpack.dev.config.js#L8) here?

```suggestion
import { Factory } from 'rosie';

import messages from '../../../messages';
```